localization/english/event_localization/wc_draenor_events_l_english.yml

```
@@ -0,0 +1,21 @@
+ WCDRA_3_OPT_A:0 "Leave no stone unturned."
+ WCDRA_3_OPT_B:0 "Two worlds will stay connected."
+ 
+ WCDRA_5_DESC:0 "All the world trembles in the wake of this momentous day: the Dark Portal has been destroyed.\n\nBy bringing the best and the brightest of [portal_destroyer.GetFullNamePossessive] spellcasters to bear, the great stone gateway that served as a bridge between Azeroth and Draenor was torn asunder and obliterated to nothing more than a pile of rubble and an ominous rift; a tear in the fabric of space and time itself. As the dark energies of the portal exploded outward, the Black Morass - already corrupted by the Portal’s influence - has been scorched into an arid waste of dead earth that many are now beginning to call the Blasted Lands. No one is sure whether this damage is irreversible or if the land might someday be healed, and many magical masters are concerned by what further damage the rift might cause.\n\nAn even greater mystery, however, is what fate yet has in store for the people of Draenor who were caught on either side of the portal during its destruction. Those on Azeroth must now carve their own path in life and find a place in this world as best as they can, while those left behind on Draenor will be doomed to die a slow and painful death alongside their world if nobody stabilizes the rift and reopens the portal…\n\n[portal_destroyer.GetFullNameNicknamed] will do what they can to ensure that doesn’t happen, of course, but what can be done if there yet remains someone with the power and will to open it from the other side?"
+ WCDRA_5_OPT_A:0 "We used to call it home..."
+ WCDRA_5_OPT_B:0 "What a loss."
+ WCDRA_5_OPT_C:0 "And may it never open again."
+ 
+ WCDRA_100_TITLE:0 "Take It on the Other Side"
+ WCDRA_100_DESC:0 "Your guards bring you a [refugee.Custom('WomanManGirlBoy')]. They tell [refugee.GetSheHe] was caugh with a group on a patrol near the [refugee.Custom('GetDarkPortalRift')]. All the evidence speaks that they somehow got to the [refugee.Custom('GetPortalRift')] unnoticed and crossed it.\n\nWhen you ask the caught, [refugee.GetSheHe] tells you [refugee.GetSheHe] had no other option but to step in the [refugee.Custom('GetPortalRift')]."
+ WCDRA_100_DESC_draenei:0 "The draenei are brought to extinction by the Orcish Horde. Out of despair, [refugee.GetHerHis] group decided to get to the other side looking for a better place.\n\n"
+ WCDRA_100_DESC_arakkoa:0 "Civil and religious wars took many lives and the rest was slain by the Orcish Horde. [refugee.GetHerHis|U] brothers and sisters decided to get to the other side looking for a better place.\n\n"
+ WCDRA_100_DESC_village:0 "The soil has lost its fertility, so [refugee.GetHerHis] village is starving. Those who still can walk decided to get to the other side looking for a better place.\n\n"
```

Minor change. Not completely necessary, but I think it is better this way.
```suggestion
 WCDRA_100_DESC_village:0 "The soil has lost its fertility, so [refugee.GetHerHis] village is starving. Those who still can walk decided to get to the other side, hoping to find a better place.\n\n"
```

---

@NovelConcept tweaked the text a lot.